server.py

```diff
@@ -550,6 +554,42 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        pass
+
+    def put(self, region):
+        print "entering merge_put"
+        dao = Dao(region, mongo_client=mongo_client)
+        requesting_user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(requesting_user, region): #wow, such auth
```

Comment:
LOL good comment. I think what I wanted here "if the player is an admin for ANY region". can you change the check to `if request_user.admin_regions`?